.github/workflows/main.yml

```diff
@@ -0,0 +1,71 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build pdf
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on:
+  push:
+    branches: [ main ]
+
```

Comment:
It would be nice to add an option to the `pull_request` as well, you can also say that the `push` option is triggered on file changes in a specific path or by extension
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths